source/_integrations/august.markdown

```diff
@@ -122,3 +122,26 @@ Using the lock operation sensors, you can detect when a user operates a lock and
 ```
 
 {% endraw %}
+
+## Token Expiration Handling
+
+The August lock access_token is stored in a file called august.conf. When it expires, the integration fails, and you see errors in the home-assistant.log file.   For example:
+
+{% raw %}
+
+```
+2020-10-02 07:47:12 ERROR (MainThread) [august.authenticator_async] Token has expired.
+2020-10-02 07:47:14 ERROR (MainThread) [homeassistant.components.august] Access token is no longer valid
+```
+
+{% endraw %}
+
+
+Here are the steps to get a new access token (tested on Home Assistant Core `0.114.4`):
+
+1. Delete the august.conf file.
+1. Restart Home Assistant.
+1. Check your email to find a message from August that contains a verification code.
+1. When the Home Assistant UI reconnects, check the Notifications panel for a message.  Open the message, choose Configure, and enter the verification code.
```

Comment:
This should be reported as a bug and not documented like this.

August uses a configuration flow, which should be retriggered when the tokens are expired (re-auth).